### ISSUE TITLE
Where are the Groovy communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/groovy#113]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Groovy supported by one or more large organizations?
* Does Groovy have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Groovy? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Groovy? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Groovy? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Groovy?
* Are there popular newsletters for Groovy?
* Is Groovy taught at programming bootcamps? (If so, what are some examples?)
* Is Groovy taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Groovy?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by KrunoGajger** [comment]
Hi Katrina,
still no answer?

Did you manage some other sources in order to get your answers?

I would love to see some of the answers you asked...

Kruno

**Comment by kytrinyx** [comment]
@KrunoGajger Yeah, I haven't heard anything about this one yet.

**Comment by KrunoGajger** [comment]
Pitty…

Did you manage to get answers on your questions on some other way?

I am new in Groovy and I would like to get as much as possible info on the beginning …

What about you? Are you programming in Groovy?

All the best,

Kruno

Sent from Mail for Windows 10

**Comment by kytrinyx** [comment]
I have not done the research. I am one of the main maintainers of Exercism, and we support 60 or so programming languages. We have the same questions for all the language tracks.

If you do the research, please do share what you find!

**Comment by KrunoGajger** [comment]
I have to do something with Groovy. I find some blogs, some people talking about it on several places on Internet… I will have something in the while…

What is Exercism and where I can find something what you guys did for the other languages?

Kruno

Sent from Mail for Windows 10

**Comment by kytrinyx** [comment]
Exercism is a site for people to ramp up in new programming languages: https://exercism.io

**Comment by Dispader** [comment]
I'll throw some words up, in the hopes that it might help folks more important than myself to weigh in.

- "Is Groovy supported by one or more large organizations?"
  - [The Apache Foundation](https://groovy-lang.org/support.html)
- "Does Groovy have an official community manager?"
   - I'm not sure if the question refers to an individual, or the parent organization?  It's biggest advocate since Pivotal stopped officially having a Developer Advocate is probably [Paul King](https://github.com/paulk-asert)
- "Do you know of specific communities (online or offline) that are enthusiastic about Groovy?"
   - [There are several](http://groovy-lang.org/usergroups.html), the ones I've been most familiar with are local to Minneapolis, *Groovy Users of Minnesota* and *Gr8Ladies*.
- "Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Groovy? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)"
   - I'm familiar with [Gr8Ladies](https://www.meetup.com/Gr8Ladies/)
- "Are there specific groups or programs dedicated to mentoring people in Groovy?"
   - Beyond individual interest groups, I'm not familiar.
- "Are there popular newsletters for Groovy?"
   - The text newsletter stopping issuing.  I'm aware of [a podcast](https://groovypodcast.podbean.com/).
- "Is Groovy taught at programming bootcamps?"
   - Several places, usually in combination with Grails or Gradle.
- "Is Groovy taught at universities?"
   - I'm not sure.

 Anyhow, hope this helps a little.

**Comment by amscotti** [comment]
Just to add to "Are there popular newsletters for Groovy?" 
Groovy Calamari from Sergio del Amo (@sdelamo) - http://groovycalamari.com/
Along with a https://podcast.groovycalamari.com/index.html

**Comment by kytrinyx** [comment]
@Dispader @amscotti thanks! This is helpful!
